msgcntr/messageforums-app/src/webapp/jsp/dfReviseTopicSettingsAttach.jsp

```diff
@@ -44,7 +44,50 @@
 	function closeDateCal(){
 		NewCal('revise:closeDate','MMDDYYYY',true,12, '<h:outputText value="#{ForumTool.defaultAvailabilityTime}"/>');	
 	}
+
+	// locate and record SELECT dom nodes with Contributor OPTIONs initially SELECTED
+	function primeAutoCreate() {
+		if (window.originalContributors) {
+			return; // prime only once
+		}
+		window.originalContributors = [];
+
+		// detect which SELECTs will have their value saved.  do detection only once, on page load ...
+		$("span.permissionRow > span > select").each(function() {
+			if ( $(this).children("option:selected[value='Contributor']").length === 1 ) {
```

Comment:
Just a note that 'Contributor' (and 'None' below) does not need to be internationalized, as the `value` attributes are always in English; it's just the display value that's translated:

![screenshot_20180303_131730](https://user-images.githubusercontent.com/10403943/36937696-28e16956-1f0f-11e8-800f-acddb73a89ed.png)